RealismClient/Sprint.client.lua

```diff
@@ -0,0 +1,33 @@
+------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
+-- Hedreon, 2022
+-- Realism Sprint
+------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
+
+local UserInputService = game:GetService("UserInputService")
+local TweenService     = game:GetService("TweenService")
+local Players          = game:GetService("Players")
+
+local TweenInformation = TweenInfo.new(0.25, Enum.EasingStyle.Sine, Enum.EasingDirection.InOut, 0, false, 0)
+local CurrentCamera    = workspace.CurrentCamera
+local LocalPlayer      = Players.LocalPlayer
+local Character        = LocalPlayer.Character or LocalPlayer.CharacterAdded:Wait()
+
+script.Parent = Character
+
+UserInputService.InputBegan:Connect(function(Input)
+	if Input.UserInputType == Enum.UserInputType.Keyboard then
```

Comment:
What about for mobile/controller?

Reply:
fine ill do that

Reply:
> What about for mobile/controller?

oh wait, I actually have the same script but with full mobile support done already but I'm not going to do controller support since I don't have an xbox. only phone and PC.

Reply:
> What about for mobile/controller?

Done.